arviz/data/io_numpyro.py

```diff
@@ -132,12 +132,12 @@ def posterior_to_xarray(self):
     @requires("posterior")
     def sample_stats_to_xarray(self):
         """Extract sample_stats from NumPyro posterior."""
-        # match PyMC3 stat names
+        # match arviZ stat names
```

Comment:
I think we can delete the comment now, all libraries will have some kind of renaming to match arviz convention.